src/wifi/os_adapter.rs

```diff
@@ -1145,19 +1146,16 @@ pub unsafe extern "C" fn wifi_rtc_disable_iso() {
  * Name: esp_timer_get_time
  *
  * Description:
- *   Get system time of type int64_t
- *
- * Input Parameters:
- *   periph - No mean
+ *   Get time in microseconds since boot.
  *
  * Returned Value:
- *   System time
+ *   System time in micros
  *
  ****************************************************************************/
 #[no_mangle]
 pub unsafe extern "C" fn esp_timer_get_time() -> i64 {
     trace!("esp_timer_get_time");
-    (crate::timer::get_systimer_count() / crate::timer::TICKS_PER_SECOND / 1_000) as i64
```

Comment:
Is this correct by the way? Can't the "time since boot" actually get lower when the systimer wraps around?

Maybe we need a proper rolling timer instead (don't worry about it for this PR though)

Reply:
it's probably not really good this way - I think we have a few places where the wrapping isn't correctly handled - I should create an issue for that 🤔

Reply:
created issue #104 for that